charts/incubator/meshcentral/Chart.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: meshcentral
+version: 0.0.1
+appVersion: "1.0.0"
+description: MeshCentral is a full computer management web site
+type: application
+deprecated: false
+home: https://github.com/truecharts/apps/tree/master/charts/stable/meshcentral
+icon: https://truecharts.org/_static/img/appicons/meshcentral.png
+keywords:
+  - meshcentral
+  - teamviewer
+  - rdp
+sources:
+  - https://github.com/Ylianst/MeshCentral
+dependencies:
+- name: common
+  repository: https://library-charts.truecharts.org
+  version: 9.1.15
```

You're using a common version without mongodb support, that is causing your issues.

```suggestion
- name: common
  repository: https://library-charts.truecharts.org
  version: 9.2.3
```